Wordle/Wordle/GameManager.cpp

```diff
@@ -0,0 +1,155 @@
+#include "GameManager.h"
+#include "FileIO.h"
+#include "DateTime.h"
+
+#include <iostream>
+#include <string>
+#include <cstdlib>
+#include <ctime>
+#include <vector>
+
+
+const unsigned sizeArray = 100;
+
+std::string getRandomWord()
+{
+    //get an array of words
+	std::string listWords[sizeArray];
+    readListWords("fiveLetterWords.txt", listWords, sizeArray);
```

Comment:
Assuming there're exactly 100 words in the file every time is not too flexible. The point to use files at all is to split code from data so you can modify them separately. For example, just update content and number of words in data without re-releasing your app. 
So, it's better to make it more flexible, for example by making the number of words a part of file and allocate memory for the array at runtime. Or just read them all and allocate more memory if needed (using std::vector or using new by yourself)